compiler/rustc_codegen_llvm/src/coverageinfo/llvm_cov.rs

```diff
@@ -2,8 +2,9 @@
 
 use std::ffi::CString;
 
+use rustc_llvm::ffi;
```

Comment:
Maybe all these places that previously used `coverageinfo::ffi` should rename the new path, so they don't need so much code churn otherwise:
```suggestion
use rustc_llvm::ffi::coverageinfo as ffi;
```